src/App.js

```diff
@@ -31,8 +43,30 @@ function App() {
     }
   }, [todoList, isLoading]);
 
-  function addTodo(newTodo) {
-    setTodoList([...todoList, newTodo])
+  async function addTodo(newTodo) {
```

Comment:
@JoHeyJo .. Thanks a lot for all the suggestions and helping me to write cleaner code :). I have worked as you suggested. It would be great if you can take a final look and approve the changes.